src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -3630,8 +3652,8 @@ namespace EvaporativeCoolers {
         // Using/Aliasing
         using BranchNodeConnections::SetUpCompSets;
         using DataGlobals::NumOfZones;
-        using DataHVACGlobals::FanType_SimpleConstVolume;
-        using DataHVACGlobals::FanType_SimpleOnOff;
+        // using DataHVACGlobals::FanType_SimpleConstVolume;
+        // using DataHVACGlobals::FanType_SimpleOnOff;
```

Comment:
If these are other changes needed, these should be deleted.

Reply:
Done.